cmd/influxd/config.go

```diff
@@ -251,6 +246,38 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Address       string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          uint16 `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeparator string `toml:"name-separator"`
+}
+
+// Default carbon port per http://graphite.readthedocs.org/en/1.0/feeding-carbon.html
+const defaultGrahitePort = 2004
+
+// ConnnectionString returns the connection string for this Graphite config in the form
+// host:port.
```

Comment:
nit: can you move the dangling `host:port` up to the previous line? It seems awkward here.